docs/experiments.rst

```diff
@@ -91,14 +91,57 @@ FB15K-237
                                                          seed: 0;
                                                          batches_count: 300;
 
+  ConvE    1060     0.26   0.19     0.28     0.38       k: 200;
+                                                         epochs: 4000;
+                                                         loss: bce;
+                                                         loss_params: {label_smoothing=0.1}
+                                                         optimizer: adam;
+                                                         optimizer_params:
+                                                         lr: 0.0001;
+                                                         embedding_model_params:{
+                                                         conv_filters: 32,
+                                                         conv_kernel_size: 3,
+                                                         dropout_embed: 0.2,
+                                                         dropout_conv: 0.1,
+                                                         dropout_dense: 0.3,
+                                                         use_batchnorm: True,
+                                                         use_bias: True};
+                                                         seed: 0;
+                                                         batches_count: 100;
+
+  ConvE*     234      0.32   0.23     0.35     0.50       k: 200;
```

Comment:
Should we call ConvE*->ConvE and vice versa. Since the current conve* is the original paper implementation?